includes/templates/bootstrap/templates/tpl_products_next_previous.php

```diff
@@ -37,14 +37,14 @@
 </div>
 
 <div class="btn-group d-block d-sm-none" role="group">
-<a class="p-1" href="<?php echo zen_href_link(zen_get_info_page($previous), "cPath=$cPath&products_id=$previous"); ?>"><span class="btn btn-primary"><?php echo '<i class="fas fa-angle-left"></i>'; ?></span></a>
+<a class="p-1" href="<?php echo zen_href_link(zen_get_info_page($previous), "cPath=$cPath&products_id=$previous"); ?>" aria-label="Prev"><span class="btn btn-primary"><?php echo '<i class="fas fa-angle-left"></i>'; ?></span></a>
```

Comment:
Shouldn't the "Prev" and "Next" elements be language constants?

Reply:
Probably but there are no defined constants for next and previ.